### ISSUE TITLE
Upgrading to `62.9.0` produces incorrect handling of `=` string?

### ISSUE THREAD

**Issue by moltar** [gajus/eslint-plugin-jsdoc#1685]
After upgrading to `v62.9.0`, the following diff is produced:

```diff
     /**
-     * @see https://us-east-1.console.aws.amazon.com/acm/home?region=us-east-1#/certificates/UUID
+     * @see https://us-east-1.console.aws.amazon.com/acm/home?region
      */
```

Thanks!

**Comment by brettz9** [comment]
What is your config or the rule which triggers if you don't auto-fix?

**Comment by moltar** [comment]
Ah, good question & interesting, as it appears to be reported by the prettier plugin:

```
  41:69  error  Delete `=us-east-1#/certificates/xxx`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
```

But it occurs only during this plugin's upgrade, not on the mainline branch.

**Comment by brettz9** [comment]
That version update @es-joy/jsdoccomment which upgraded `comment-parser` which had this probably related PR: https://github.com/syavorsky/comment-parser/pull/189/changes .

However, I can't think of what rule would do this. There is `jsdoc/no-defaults`, but that doesn't apply to `see` tags. If prettier is reporting it, perhaps you should report it to them, as it is being reported as a prettier error.

**Comment by moltar** [comment]
I understand that it is reported by prettier. But it only happens on this upgrade on several repos.

**Comment by moltar** [comment]
Anything I can do to help debug this better?

**Comment by brettz9** [comment]
I know that it only happens on this upgrade, but we need to get prettier to inform us of which rule is responsible. Per AI, it appears you can do this by using `eslint-plugin-prettier`. If you follow its instructions, perhaps you can run `npx eslint .` and get the specific rule of ours that is being violated.

**Comment by moltar** [comment]
Investigated this in depth — confirming @brettz9's hunch about [comment-parser#189](https://github.com/syavorsky/comment-parser/pull/189). Posting in case it helps close this issue out, since the actual culprit is not a rule in this plugin.

### Root cause

`comment-parser@1.4.6` (released 2026-03-28, via PR #189 closing #186) extended `nameTokenizer` to also split **non-optional** names on `=` and assign the right-hand side to `default`. The intent was `@property {number} BITMASK_VALUE_A=16`, but the rule applies to any name containing `=` — including URLs.

Filed upstream: https://github.com/syavorsky/comment-parser/issues/190

### What actually mangles the comment

Not this plugin. It's **`prettier-plugin-jsdoc@1.8.0`**, which:

1. Calls `comment-parser`'s `nameTokenizer()` on `@see https://example.com/foo?bar=baz#section`.
2. With 1.4.6, that returns `{ name: 'https://example.com/foo?bar', default: 'baz#section', optional: false }`.
3. `prettier-plugin-jsdoc`'s `stringify` for `@see` emits only `name` (not `default` — `default` is only emitted via `assignOptionalAndDefaultToName` when `optional` is true), so `default` is silently dropped on output.

Reproducible standalone (no `eslint-plugin-jsdoc` involved):

```js
import * as prettier from 'prettier'
import * as jsdoc from 'prettier-plugin-jsdoc'

const src = `/**\n * @see https://example.com/foo?bar=baz#section\n */\nexport const x = 1\n`
console.log(await prettier.format(src, { parser: 'typescript', plugins: [jsdoc] }))
// → /** @see https://example.com/foo?bar */
```

Pinning `comment-parser` to `1.4.5` via a pnpm/npm override restores correct output. Confirmed by swapping the resolved version in `node_modules` and re-running.

### Why upgrading this plugin triggered it

`eslint-plugin-jsdoc@62.9.0` updated `@es-joy/jsdoccomment`, which bumped its `comment-parser` range. With pnpm dedupe, that lifted the shared `comment-parser` resolution to `1.4.6`, which `prettier-plugin-jsdoc` then picked up via `^1.4.0`. So the upgrade was the trigger via lockfile movement, not via any rule in this plugin actually transforming `@see`.

Happy to close this on my end — the actionable trail leads to `syavorsky/comment-parser#190` and (optionally) a follow-up at `hosseinmd/prettier-plugin-jsdoc`.

**Comment by moltar** [comment]
Closing as not applicable to this repo! 😁
